data/pages/transparency.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Transparency
+summary: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+image: /static/images/avatar.png
+layout: PageLayout
+---
+
+As a 501(c)(3) public charity, OpenSats is committed to upholding high
+transparency, integrity, and legal compliance standards. As an organization that
+is embedded in the free and open-source movement, we cherish the principles of
+transparency, openness, as well as conversing and working in public.
+
+By adhering to these principles, we aim to demonstrate our commitment to
+responsible sat management, good corporate citizenship, and the long-term
```

Comment:
"sat management" - I assume this means bitcoin treasury management. If that's the case, I would prefer to use a more explicit term such as "bitcoin treasury management" or "treasury management" or "donation management".
My concern is that, for now, some or many people may not understand what "sat management" means.